graylog2-server/src/main/java/org/graylog2/rest/resources/streams/StreamResource.java

```diff
@@ -325,6 +331,14 @@ public Response cloneStream(@ApiParam(name = "streamId", required = true) @PathP
             streamService.addAlertCondition(stream, alertCondition);
         }
 
+        for (AlarmCallbackConfiguration alarmCallbackConfiguration : alarmCallbackConfigurationService.getForStream(sourceStream)) {
```

Comment:
What happens if the clone/copy process fails for one of the alarm callbacks and an exception is being thrown?